project-base/app/src/DataFixtures/Demo/SettingValueDataFixture.php

```diff
@@ -117,7 +117,7 @@ public function load(ObjectManager $manager)
             );
             $this->setting->setForDomain(
                 SeoSettingFacade::SEO_ROBOTS_TXT_CONTENT,
-                'Disallow: *?filter=',
+                "Disallow: *?filter=\n\nUser-agent: *\nCrawl-delay: 3\nRequest-rate: 300/1m",
```

Comment:
This should be mentioned in UPGRADE notes (see #project-base-diff...)

Reply:
you are right, fixed